orbit/pkg/table/extension_darwin.go

```diff
@@ -25,7 +26,7 @@ import (
 	"github.com/osquery/osquery-go/plugin/table"
 )
 
-func PlatformTables() []osquery.OsqueryPlugin {
+func PlatformTables(r *Runner) []osquery.OsqueryPlugin {
```

Comment:
Same question as above. 
I can only see that we use the osquery path and not sure why we pass the whole Runner object.